console/program/src/data/identifier/parse.rs

```diff
@@ -70,14 +70,8 @@ impl<N: Network> Debug for Identifier<N> {
 impl<N: Network> Display for Identifier<N> {
     /// Prints the identifier as a string.
     fn fmt(&self, f: &mut Formatter) -> fmt::Result {
-        // Convert the identifier to bits.
-        let bits_le = self.0.to_bits_le();
-
-        // Convert the bits to bytes.
-        let bytes = bits_le
-            .chunks(8)
-            .map(|byte| u8::from_bits_le(byte).map_err(|_| fmt::Error))
-            .collect::<Result<Vec<u8>, _>>()?;
+        // Convert the identifier to bytes.
+        let bytes = self.0.to_bytes_le().map_err(|_| fmt::Error)?;
```

Comment:
it didn't seem like there was a reason for the twofold conversion here, but feel free to revert if I missed something

Reply:
I think the concern may have been that they might not be equivalent? Can you confirm this?

Reply:
test case added in https://github.com/AleoHQ/snarkVM/pull/1884/commits/bdad7614ccc3cacc024bfe61e1937c322407f7e2; as anticipated, both implementations are equivalent :+1: